comp/api/api/apiimpl/internal/agent/agent.go

```diff
@@ -223,7 +227,11 @@ func getStatus(w http.ResponseWriter, r *http.Request, statusComponent status.Co
 	}
 	w.Header().Set("Content-Type", contentType)
 
-	s, err := statusComponent.GetStatus(format, verbose)
+	if section, ok := section.Get(); ok {
+		s, err = statusComponent.GetStatusBySection(section, format, verbose)
```

Comment:
A few points:
- We should normalize the section provided by the user and the section name store in the status component. That way the user do not have to match the section in a case sensitive maner. Ex `status component Forwarder` should work and return the forwarder section.
- Another nice improvement would be to list the available sections name if the section provided by the customer do not exists. It might require returning a custom error and extending the status component interface 😄

Reply:
Good ideas !

Reply:
Good ideas !
- case insensitive search done in 4bf244acd7731c6e9dc180784bc59c014b753eb2
- return available component done in 1d4063f9beea7b118bff1c86488e13848ea1c68d